src/validation_provider/index.ts

```diff
@@ -20,28 +21,31 @@ export class ValidationProvider implements Disposable {
           if (!e.affectsConfiguration("dbt")) {
             return;
           }
-          this.validateCredentials();
+          this.validateCredentials(false);
         },
         this,
         this.disposables,
       ),
     );
   }
 
-  private async validateCredentials() {
-    const config = this.altimate.getConfig();
-    if (!config) {
-      return;
-    }
-    const { key, instance } = config;
+  async validateCredentials(silent: boolean) {
```

Comment:
Create method alias for silent mode, booleans are difficult to understand in the calling code.